rust/ql/test/library-tests/dataflow/models/models.ext.yml

```diff
@@ -1,4 +1,16 @@
 extensions:
+  - addsTo:
+      pack: codeql/rust-all
+      extensible: sourceModel
+    data:
+      - ["repo::test", "crate::enum_source", "ReturnValue.Variant[crate::MyFieldEnum::D::field_d]", "test-source", "manual"]
+      - ["repo::test", "<crate::MyFieldEnum>::source", "ReturnValue.Variant[crate::MyFieldEnum::C::field_c]", "test-source", "manual"]
```

Comment:
What does the `repo` prefix mean, and why do we need the `crate::` prefix?

Reply:
How would this work if there are items from multiple crates involved?

Reply:
Ah, right. I decided to not include the crate origins in the `Variant` arguments, since that would bloat things. But you are right that this can potentially give rise to some ambiguity, if two crates define variants with the same canonical paths.